src/control/Control.Layers.js

```diff
@@ -208,6 +208,8 @@ export var Layers = Control.extend({
 
 		if (!collapsed) {
 			this.expand();
+
+			link.setAttribute('aria-label', Layers);

```

Comment:
@whtann Note that this sets the `aria-label` on the `<a>` control. That's not what is asked for in https://github.com/Leaflet/Leaflet/issues/7529:

> When the layer control's `collapsed` option is set to `false` _the [`leaflet-control-layers-list`](https://github.com/Leaflet/Leaflet/blob/3466cbba34351ee0d7477a83eb0215cc0e6b454b/src/control/Control.Layers.js#L184) element should have `aria-label="Layers"`_.

(again, this is because in that scenario the `<a>` is not available to provide the necessary context)